luatest/server.lua

```diff
@@ -1,29 +1,42 @@
---- Class to run tarantool instance.
+--- Class to manage Tarantool instances.
 --
 -- @classmod luatest.server
 
 local checks = require('checks')
+local clock = require('clock')
+local digest = require('digest')
+local errno = require('errno')
+local fiber = require('fiber')
+local fio = require('fio')
 local http_client = require('http.client')
 local json = require('json')
 local log = require('log')
 local net_box = require('net.box')
+local uri = require('uri')
 local _, luacov_runner = pcall(require, 'luacov.runner') -- luacov may not be installed
 
+local assertions = require('luatest.assertions')
 local HTTPResponse = require('luatest.http_response')
 local Process = require('luatest.process')
 local utils = require('luatest.utils')
 
--- Linux uses max 108 bytes for Unix domain socket paths, which means a 107 characters string ended by a null
--- terminator. Other systems use 104 bytes and 103 characters strings.
-local max_unix_socket_path = {linux = 107, other = 103}
+local DEFAULT_VARDIR = '/tmp/t'
+local DEFAULT_ALIAS = 'server'
+local DEFAULT_INSTANCE = fio.pathjoin(
+    fio.dirname(fio.abspath(package.search('luatest.server'))), 'server_instance.lua'
+)
+local WAIT_TIMEOUT = 60
+local WAIT_DELAY = 0.1
 
 local Server = {
     constructor_checks = {
-        command = 'string',
-        workdir = 'string',
+        command = '?string',
```

Comment:
Order of variables here does not match the order in the constructor docstrings, lines 63-94

Reply:
I decided not to do this because it can be considered as not relevant changes.